### PR TITLE
Lear 775 | API Testing

### PR DESCRIPTION
**Pull request by Jbholae**
Solving error regarding API testing

### CODE REVIEW

File: api/controllers/posts.go [hunk 1]
```diff
@@ -66,15 +67,29 @@ func (cc PostsController) CreatePosts(c *gin.Context) {
 // UpdatePosts -> Update Post
 func (cc PostsController) UpdatePosts(c *gin.Context) {
 	trx := c.MustGet(constants.DBTransaction).(*gorm.DB)
+	id, err := strconv.Atoi(c.Param("id"))
```
Comment (sawal-timsina) [inline api/controllers/posts.go]:
```suggestion
	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
```

File: api/controllers/posts.go [hunk 2]
```diff
@@ -130,13 +146,23 @@ func (cc PostsController) PostLikes(c *gin.Context) {
 // DeletePosts -> Delete Posts
 func (cc PostsController) DeletePosts(c *gin.Context) {
 	id, err := strconv.Atoi(c.Param("id"))
```
Comment (sawal-timsina) [inline api/controllers/posts.go]:
```suggestion
	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
```

File: api/repository/follow.go [hunk 3]
```diff
@@ -33,8 +33,6 @@ func (c FollowRepository) GetFollowerCount(ID string) (count int, err error) {
 }
 
 func (c FollowRepository) GetFollowingCount(ID string) (count int, err error) {
-	return count, c.db.DB.Model(&models.Follower{}).Select("COUNT(follow_user_id)").Where("user_id = ?", ID).Find(&count).Error
-}
```
Comment (sawal-timsina) [inline api/repository/follow.go]: @Jbholae did you removed it knowingly?

Comment (Jbholae) [reply api/repository/follow.go]: @sawal-timsina No, I am unaware about this.

Comment (sawal-timsina) [reply api/repository/follow.go]: @Jbholae Let's rollback this line. We need this as well.

Comment (Jbholae) [reply api/repository/follow.go]: @sawal-timsina  As I've checked, this line of code is still there

Comment (sawal-timsina) [reply api/repository/follow.go]: @Jbholae its been removed from line 36 🙇 Please check on your branch 
Screen Shot 2022-09-12 at 3 30 35 PM

**Review by sawal-timsina — APPROVED** [review]
LGTM

**Pull request merged**